sungc/deglint.py

```diff
@@ -249,37 +313,59 @@ def get_band_list(self):
 
         Returns
         -------
-        bandList : list
+        bandlist : list
             A list of pathlib
 
-        bandNums : list of strings
+        bandnums : list of strings
             A list of band numbers
 
-        bandNames : list of string
+        bandnames : list of string
             A list of band names
+
+        Raises:
+            Exception if any of the bands do not exist
         """
-        bandList = []
-        bandNums = []
-        bandNames = []
+        # a list of bands that are not needed for deglinting
+        skip_bands = [
+            "contiguity",
+            "azimuthal_exiting",
+            "azimuthal-incident",
```

Comment:
just checking that the `-` and `_` discrepancy is correct

Reply:
You're right, it's meant to be "azimuthal-exiting"